src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -906,6 +907,10 @@ namespace ReportSizingManager {
 									TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
 								} else {
 									TotCapTempModFac = 1.0;
+									
+									if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
+										EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, TotCapTempModFac);
```

Comment:
You will need to add an if block for CoilVRF_FluidTCtrl_Cooling and do the same thing there.